docs/api/app_info.rst

```diff
@@ -0,0 +1,84 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Application Info
+================
+
+This section documents everything related to :class:`Client`'s application information.
```

Comment:
These aren't just for Client, some of these classes can show app info for other apps.

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)